TaskSharper.DataAccessLayer.Google/Calendar/Service/GoogleCalendarService.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Google.Apis.Calendar.v3;
+using Google.Apis.Services;
+using TaskSharper.Domain.Calendar;
+
+namespace TaskSharper.DataAccessLayer.Google.Calendar.Service
+{
+    public class GoogleCalendarService : BaseService, ICalendarService
+    {
+        private readonly CalendarService _service;
+        public GoogleCalendarService(CalendarService service = null)
```

Comment:
Why is CalendarService nullable.? Aren't we highly dependent on it. Consider making it not nullable, and therby forcing the user to create it.

Reply:
Didn't make sense for the user to create it. Changed the parameter to string applicationName.